ruby/CRUD-examples/delete/extra_fun_1.rb

```diff
@@ -0,0 +1,42 @@
+# Code Sample from the tutorial at https://learncodeshare.net/2016/11/09/delete-crud-using-ruby-oci8/
+#  section titled "Extra Fun 1"
+# Using the base template, the example code executes a simple delete using named bind variables.
+
+require 'oci8'
+connectString = ENV['DB_CONNECT'] # The environment variable for the connect string: DB_CONNECT=user/password@database
+
+def get_all_rows(label, data_type = 'people')
+  connectString = ENV['DB_CONNECT'] # The environment variable for the connect string: DB_CONNECT=user/password@database
```

Comment:
Why is the connect string retrieved twice? The location of the first retrieval indicates that the connect string is global but then it is retrieved again. Shouldn't either that function accept a connection as input parameter or the connect string be retrieved just before line 32 to make this clearer to the reader?

This applies multiple times throughout the code snippets.